<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>450</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>571</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>571</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Cole - OoT Mod Helper</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>logo.png</normaloff>logo.png</iconset>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>-4</x>
      <y>0</y>
      <width>581</width>
      <height>451</height>
     </rect>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="documentMode">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="actorTab">
     <attribute name="title">
      <string>Actors</string>
     </attribute>
     <widget class="QGroupBox" name="paramGroup">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>10</y>
        <width>311</width>
        <height>411</height>
       </rect>
      </property>
      <property name="title">
       <string>Parameters</string>
      </property>
      <widget class="QComboBox" name="actorTypeList">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>20</y>
         <width>221</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="actorTypeLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Actor Type</string>
       </property>
      </widget>
      <widget class="QWidget" name="formLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>291</width>
         <height>201</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="paramLayout"/>
      </widget>
      <widget class="QCheckBox" name="ignoreTiedBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>250</y>
         <width>121</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Ignore Tied Params</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="paramBox">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>290</y>
         <width>221</width>
         <height>21</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="rotXBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>319</y>
         <width>221</width>
         <height>21</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="rotYBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>350</y>
         <width>221</width>
         <height>21</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="rotZBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>380</y>
         <width>221</width>
         <height>21</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="evalParamBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>270</y>
         <width>91</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Eval Params</string>
       </property>
      </widget>
      <widget class="QPushButton" name="paramLabel">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>290</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Parameters</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="rotXLabel">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>320</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Rot X</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="rotYLabel">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>350</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Rot Y</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="rotZLabel">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>380</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Rot Z</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>310</y>
        <width>221</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>Options</string>
      </property>
      <widget class="QPushButton" name="openActorFileBtn">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Open Actor File</string>
       </property>
      </widget>
      <widget class="QPushButton" name="saveActorFileBtn">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>20</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Save Actor File</string>
       </property>
      </widget>
      <widget class="QPushButton" name="deleteActorBtn">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Delete Actor</string>
       </property>
      </widget>
      <widget class="QPushButton" name="addActorBtn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Add Actor</string>
       </property>
      </widget>
      <widget class="QPushButton" name="editActorBtn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>50</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Edit Actor</string>
       </property>
      </widget>
      <widget class="QPushButton" name="deleteAllBtn">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>80</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Delete All</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="searchGroup">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>221</width>
        <height>291</height>
       </rect>
      </property>
      <property name="title">
       <string>Search Actor</string>
      </property>
      <widget class="QLineEdit" name="searchBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>201</width>
         <height>21</height>
        </rect>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="whatsThis">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Search...</string>
       </property>
      </widget>
      <widget class="QLabel" name="actorFoundLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>270</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Found: 0</string>
       </property>
      </widget>
      <widget class="QComboBox" name="actorCategoryList">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>250</y>
         <width>201</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QListWidget" name="actorFoundBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>201</width>
         <height>192</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="flagTab">
     <attribute name="title">
      <string>Flags</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="statTab">
     <attribute name="title">
      <string>Statistics</string>
     </attribute>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
